config.py

```diff
@@ -63,6 +64,7 @@ class Test(Config):
     ES_ENABLED = False
     SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/digitalmarketplace_test'
     DM_API_AUTH_TOKENS = 'myToken'
+    DM_API_CALLBACK_AUTH_TOKENS = 'myCallbackToken'
```

Comment:
In Development the token is 'myToken' as for DM_API_AUTH_TOKENS, does it need to be different here?

Reply:
Nah. I think it'll be easier if the token is the same for developers so they don't have to remember two different tokens. But for tests, we need to differentiate in order to assert that each kind of token only works for its part of the API.

I'm happy to update the dev token to `myCallbackToken` if you think it's useful for devs to understand and always be aware that these are two different tokens.